client/api.c

```diff
@@ -1524,15 +1526,23 @@ TDNFRepoQuery(
     dwError = SolvGetQueryResult(pQuery, &pPkgList);
     BAIL_ON_TDNF_ERROR(dwError);
 
-    /* handle query options */
+    if(pRepoqueryArgs->pszQueryFormat)
```

Comment:
We need to address the case someone tries to do a mixed query, like: `tdnf repoquery --qf "%{provides}" --requires tdnf `. Currently, it just ignores any other options when `--qf` is given.

It maybe okay to throw an error, but look what `dnf `does.

Reply:
In fedora when we issue mixed query than the result of last query is printed. 
In photon if we issue mixed query than the result of first query is printed.
Should we add an error for mixed query?

Reply:
Yes, I think we should throw an error. Looks like `dnf` isn't properly handling it either.

Reply:
Added code to throw error for mixed query

Reply:
Added a new error code: "ERROR_TDNF_CLI_INVALID_MIXED_QUERY_QUERYFORMAT"